src/test/java/bio/terra/workspace/integration/WorkspaceIntegrationTest.java

```diff
@@ -28,24 +29,91 @@
 @TestPropertySource("classpath:application-integration-test.properties")
 public class WorkspaceIntegrationTest {
 
-  @Autowired private WorkspaceManagerTestClient workspaceManagerTestClient;
+  // TODO: As this class grows, consider if it's worth breaking down these workspace tests into
+  //  different class files based on the the type of workspace action (Create, Get, Delete, etc).
 
+  @Autowired private WorkspaceManagerTestClient workspaceManagerTestClient;
   @Autowired private TestUtils testUtils;
-
   @Autowired private TestConfiguration testConfig;
-
   @Autowired private AuthService authService;
+  private final String TAG_NEEDS_CLEANUP = "needs-cleanup";
+  private static final Logger logger = LoggerFactory.getLogger(WorkspaceIntegrationTest.class);
+
+  // Variable for workspace Id's to clean up in tearDown(TestInfo testInfo) method, after
+  // running applicable test method(s)
+  private UUID workspaceIdToCleanup;
 
   @BeforeEach
   public void setup() {}
 
+  /*
+   * TODO: Cloud Recourse Library (CRL) is planning to broadly address resource cleanup for integration tests in MC Terra
```

Comment:
typo: "Resource" ... though "Cloud Recourse" is interesting in its own right ...

Reply:
Indeed! :)